src/main/kotlin/org/objectionary/ddr/graph/InnerPropagator.kt

```diff
@@ -61,10 +61,12 @@ class InnerPropagator(
         }
     }
 
+    /**
+     * @todo #44:30min this solution is naive, optimize it (see commented out lines)
+     */
     @Suppress("MAGIC_NUMBER")
     private fun processDecorators() {
         val repetitions = 5
-        // @todo #44:30min this solution is naive, optimize it (see commented out lines)
         // while (decorators.containsValue(false)) {
```

Comment:
@c71n93 do we really need this comment here?

Reply:
@mximp without this comment, it is not entirely clear what was meant in TODO. However, to be honest, it's still unclear with the comment. Can I fix this in the next PRs, when I need to figure out what this code does?

Reply:
@c71n93 Just move the commented snippet along with TODO